### PR TITLE
DOC-619 | ArangoGraph: scale down deployment, upgrade single server

### PR DESCRIPTION
**Pull request by nerpaula**
### Description

Added in the Edit deployment section info about:
- scaling down deployments
- upgrading Single Server deployments to OneShard or Sharded

### CODE REVIEW

**Comment by nerpaula** [comment]
> Can you only get as much disk space as is associated with a RAM config? What if you need more space than the max memory we offer (A256 AFAIK)? We used to let people configure a maximum disk size to grow to, is there still something like this that increases the storage capacity? It's not very clear from the docs.

By default, yes. If you need more, than you need to get in touch with the ArangoGraph team. There is an internal flag per-organization called `AllowCustomDiskSettings`, which defaults to `false` for all organizations. If enabled, then you can customize your disk size. But it is something on-demand AFAIK.

File: site/content/3.10/arangograph/deployments/_index.md [hunk 1]
```diff
@@ -250,20 +250,23 @@ attached to your role. Read more about [roles and permissions](../security-and-a
    - Select a different CA certificate.
    - Add or remove an IP allowlist.
 5. In the **Configuration** section, you can do the following:
-   - Upgrade the memory size per node. The disk size is automatically set for you.
+   - Upgrade or scale down the memory size per node. The disk size is
+     automatically set for you.
+     {{< warning >}}
+     When upgrading or scaling down the memory size in AWS deployments, the new
+     value gets locked and cannot be changed again until the cloud provider rate
+     limit is reset.
```
[the remark this answers is not in the document]
Comment (nerpaula) [reply site/content/3.10/arangograph/deployments/_index.md:258]: image

It's an AWS thing.

File: site/content/3.10/arangograph/deployments/_index.md [hunk 2]
```diff
@@ -250,20 +250,23 @@ attached to your role. Read more about [roles and permissions](../security-and-a
    - Select a different CA certificate.
    - Add or remove an IP allowlist.
 5. In the **Configuration** section, you can do the following:
-   - Upgrade the memory size per node. The disk size is automatically set for you.
+   - Upgrade or scale down the memory size per node. The disk size is
```
[the remark this answers is not in the document]
Comment (nerpaula) [reply site/content/3.10/arangograph/deployments/_index.md:253]: Was considering this too. Let's make it "scale up or down".

File: site/content/3.10/arangograph/deployments/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Simran-B) [reply site/content/3.10/arangograph/deployments/_index.md:258]: I suppose it refers to API throttling then

**Review by Robert-Stam — APPROVED** [review]
LGTM, I would not describe the custom disk sizing, this is something PM and Sales want to get rid of (and is there for backward compatibility only)

**Pull request merged**